src/meta/src/hummock/compaction.rs

```diff
@@ -204,7 +217,8 @@ impl CompactStatus {
                         {
                             match posterior {
                                 LevelHandler::Overlapping(_, _) => unimplemented!(),
-                                LevelHandler::Nonoverlapping(l_n_suc, _) => {
+                                LevelHandler::Nonoverlapping(compacting_ssts, _) => {
```

Comment:
Is it possible that use a different identifier here?

Reply:
renamed.